app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -20,6 +20,18 @@ def self.event_monitor_class
     ManageIQ::Providers::Openshift::ContainerManager::EventCatcher
   end
 
+  def common_logging_route_name
+    Settings.container_logging.common_logging_route
+  end
+
+  def common_logging_query
+    nil # should be empty to return all
+  end
+
+  def common_logging_path
+    '/#/dashboard/Operations-Logs-Overivew'
```

Comment:
@simon3z I agree, I changed this to '/' (default entry screen of kibana). We should find a way to inject those dashboards and then have another PR to change this accordingly.